README.md

```diff
@@ -22,12 +33,105 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
+
+        *Note*: Training scripts using GradientTape for higher-order gradients or multiple tapes are not supported. Distributed training scripts that use GradientTape are not supported at this time.
+
+   - Support `SyncOnReadVariable` in mirrored strategy - Fixes a bug that occurred because the `SyncOnRead` distributed variable was not supported with `smdebug`. Also enables the use of `smdebug` with training scripts using TF 2.x MirroredStrategy with the `fit()` API.
+
+   - Turn off hook and write only from one worker for unsupported distributed training techniques – Fixes a crash when distributed training in PyTorch framework is implemented using generic multiprocessing library, which is not a method supported by `smdebug`. This fix handles this case and ensures that tensors are saved.
+
+   - Bug fix: Pytorch: Register only if tensors require gradients – Users were observing a crash when training with pre-trained embeddings which does not need gradient updates. This fix checks if a gradient update is required and registers a backward hook only in those cases.
+
+### Setting up SageMaker Debugger
+
+`smdebug` library runs on Python 3.x. Install `smdebug` through:
+
+```
+pip install smdebug
+```
+
+#### Zero Script Change
+
+You can use your own training script while using [AWS Deep Learning Containers (DLC)](https://aws.amazon.com/machine-learning/containers/) in TensorFlow, PyTorch, MXNet and XGBoost frameworks. The AWS DLCs enable you to use Debugger with no changes to your training script by automatically adding SageMaker Debugger's `Hook`.
+The following table shows currently supported versions of the four frameworks for Zero Script Change experience.
+
+| Framework | Version |
+| --- | --- |
+| [TensorFlow](tensorflow.md) | 1.15, 2.1 |
+| [MXNet](mxnet.md) | 1.6 |
+| [PyTorch](pytorch.md) | 1.3, 1.4 |
+| [XGBoost](xgboost.md) | >=0.90-2 [As Built-in algorithm](xgboost.md#use-xgboost-as-a-built-in-algorithm)|
+
+
+#### Bring your own training container
+
+| Framework | Versions |
+| --- | --- |
+| [TensorFlow](tensorflow.md) | 1.14. 1.15, 2.0.1, 2.1.0 |
+| Keras (with TensorFlow backend) | 2.3 |
+| [MXNet](mxnet.md) | 1.4, 1.5, 1.6 |
+| [PyTorch](pytorch.md) | 1.2, 1.3, 1.4 |
+| [XGBoost](xgboost.md) | [As Framework](xgboost.md#use-xgboost-as-a-framework) |
+
+#### Support for Distributed Training and Known Limitations
+
+<table>
+    <thead>
+        <tr>
+           <th colspan=3>
+           Distributed Training
+           </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=2>Horovod</td>
+            <td>Supported</td>
+            <td>TF 1.15, PT 1.4, MX 1.6</td>
+        </tr>
+        <tr>
+            <td>Not supported</td>
+            <td>TF 2.x, PT 1.5</td>
+        </tr>
+        <tr>
+            <td>Parameter Server-based</td>
+            <td colspan=2>Not supported</td>
+        </tr>
+    </tbody>
+</table>
+
```

Comment:
hoping that we will have more contents or link to contents about DT.

Reply:
Working on it. As you provided the other PR for DT, I am reading the PR and will update this.